src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Largest Contentful Paint
```

Comment:
btw the url for this post will be web.dev/largest-contentful-paint.

Just checking that you don't want to reserve that URL for a future post, maybe for a more canonical explainer of largest contentful paint?

Reply:
The plan is for this to be that canonical post, with probably some slight tweaking to fit the style of other posts once all the metrics section is done.